app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :likes, class_name: 'Like'
+  has_many :comments, class_name: 'Comment'
+  has_many :posts, class_name: 'Post'
+
+  def three_recent_posts
+    Post.order('created_at DESC').limit(3)
+  end
```

Comment:
- [ ] This method as implemented will also return posts that don't belong to the User. Our focus is returning only the user's three most recent posts.

  When we work with Post with the capital `P` we are interacting with the global `Post` class. So getting the last three posts through the `Post` class doesn't necessarily give us only the post that belongs to the user.
  
  We need to interact with the `posts` property instead. Remember, a user has many `posts`. Therefore, `posts`, in the plural form, become a legal property we can consult within the User model.
  
  Please consider the suggestion below:
  ```suggestion
  def three_recent_posts
    posts.order('created_at Desc').limit(3)
  end
```

Reply:
This issue is well resolved! ✔️